CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+How to Contribute
+=================
+
+Getting help
+------------
+To ask questions about the XBee Library for Android go to the
+[Digi Forum](http://www.digi.com/support/forum).
+
+
+Reporting a bug
+---------------
+The way to report bugs is to use the
+[GitHub issue tracker](http://github.com/digidotcom/XBeeAndroidLibrary/issues).
+Before reporting a bug, please read the following points:
+
+1. Make sure that is really a bug by checking the
+[documentation](http://docs.digi.com/display/XBJLIB) and consulting the
+[FAQs](https://docs.digi.com/display/XBJLIB/Frequently+Asked+Questions+%28FAQs%29)
```

Comment:
Replace this link with this one: https://www.digi.com/resources/documentation/digidocs/90001438/#reference/r_xb_java_lib_faq.htm